rules/integrations/azure/initial_access_consent_grant_attack_via_azure_registered_application.toml

```
@@ -18,16 +18,70 @@ license = "Elastic License v2"
 name = "Possible Consent Grant Attack via Azure-Registered Application"
 note = """## Triage and analysis
 
-- In a consent grant attack, an attacker tricks an end user into granting a malicious application consent to access their data, usually via a phishing attack. After the malicious application has been granted consent, it has account-level access to data without the need for an organizational account.
-- Normal remediation steps, like resetting passwords for breached accounts or requiring Multi-Factor Authentication (MFA) on accounts, are not effective against this type of attack, since these are third-party applications and are external to the organization.
-- Security analysts should review the list of trusted applications for any suspicious items.
+### Investigating Possible Consent Grant Attack via Azure-Registered Application
 
+In an illicit consent grant attack, the attacker creates an Azure-registered application that requests access to data
+such as contact information, email, or documents. The attacker then tricks an end user into granting that application
+consent to access their data either through a phishing attack, or by injecting illicit code into a trusted website.
+After the illicit application has been granted consent, it has account-level access to data without the need for an
+organizational account. Normal remediation steps, like resetting passwords for breached accounts or requiring Multi-Factor
+Authentication (MFA) on accounts, are not effective against this type of attack, since these are third-party applications
+and are external to the organization.
+
+Official Microsoft Guidance around this attack can be found [here](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants).
+
+#### Possible investigation steps
+
+- From the Azure AD portal, Review the application granted permissions:
+  - Click on the `Review permissions` button on the `Permissions` blade of the application.
+  - An app should require only permissions related to the app's purpose. If that's not the case, the app might be risky.
+  - Apps that require high privileges or admin consent are more likely to be risky.
+- Investigate the app and the publisher. The following characteristics can indicate suspicious apps:
+  -  A low number of downloads.
+  -  Low rating or score or bad comments.
+  -  Apps with a suspicious publisher or website.
+  -  Apps whose last update is not recent. This might indicate an app that is no longer supported.
+- Export and examine the [Oauth app auditing](https://docs.microsoft.com/en-us/defender-cloud-apps/manage-app-permissions#oauth-app-auditing).
+To identify users affected.
```

```suggestion
To identify affected users.
```
This sentence isn't right. Not sure if it should be removed, or added to the previous line such as `Export and examine the [Oauth app auditing](...) to identify affected users.`

---

This is a continuation from the above item, just breaking the line for linting

---

But I'll fix the period that breaks it in two different phrases